### ISSUE TITLE
Add unit tests to the Vue app

### ISSUE THREAD

**Issue by Dripcoding** [TandoorRecipes/recipes#2016]
### Is your feature request related to a problem? Please describe.

Adding unit tests to the Vue app can increase confidence in future releases, prevent regressions, and catch bugs during development. These tests can be run in the ci/cd pipeline every time a PR is open and when a  PR is merged. 

### Describe the solution you'd like

Add unit tests for the basic components and the views in the Vue app using [Vue Testing Library](https://github.com/testing-library/vue-testing-library) and Jest (https://jestjs.io/).

package.json can include scripts to run tests and check code coverage locally and in a ci/cd pipeline like Github Actions.

**Comment by vabene1111** [comment]
Yes, want to do this, maybe as part of the vue 3 update or sometime after that. Will close this as I don't need an issue for it but thanks anyway
